frappe/public/js/frappe/views/reports/query_report.js

```diff
@@ -1383,6 +1383,11 @@ frappe.views.QueryReport = class QueryReport extends frappe.views.BaseList {
 			layout_direction: frappe.utils.is_rtl() ? "rtl" : "ltr",
 		});
 
+		let filter_values = [];
+		for (var key of Object.keys(applied_filters)) {
+			filter_values.push(applied_filters[key]);
```

Comment:
Filter are bit too much in filename? 

Also, Windows hates long file names. Max path is ~260 characters which can easily exceed on some reports 😅 


Filters can be added inside PDF for more info, that's a separate feature tho.

Reply:
Rest looks fine 👍

Reply:
> Filter are bit too much in filename?

Filters were added to identify the PDFs after downloading multiple ones with different filters. For example, Accounts Receivable of Customer 1, Accounts Receivable of Customer 2 etc. Opening the PDF each time to check is bit hectic :grimacing: 

> Also, Windows hates long file names. Max path is ~260 characters which can easily exceed on some reports
Agreed, maybe let's limit the length of the filename instead of removing the filters?

@ankush

Reply:
yeah, let's go with limited filenames. ~200 max.

Reply:
Updated.